html-forms/index.html

```diff
@@ -0,0 +1,49 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <title>Document</title>
+</head>
+<body>
+  <h1>Coding Form!</h1>
+  <form>
+    <label for="email">Email</label><br>
+    <input type="text" id="email"><br>
+
+    <label for="why">Why is coding cool?</label><br>
+    <select id="why">
+      <option value="Select an option">Select an option</option>
+      <option value="Because it's interesting">Because it's interesting</option>
+      <option value="Because it's fun">Because it's fun</option>
+      <option value="Because it's powerful">Because it's powerful</option>
+    </select><br><br>
+
+    <label for="language">Favorite Language(choose one)</label><br><br>
```

Comment:
This is just general text, it isn't labeling a specific form control element. You can make this a `p` element.